tests/cases/data/source/MongoDbTest.php

```diff
@@ -630,125 +537,18 @@ public function testRelationshipGenerationWithPluralNamingConvention() {
 		$this->assertEqual($expected, $result->data());
 	}
 
-	public function testCreateNoConnectionException() {
-		$db = new MongoDb(['host' => '__invalid__', 'autoConnect' => false]);
-		$this->assertException('Could not connect to the database.', function() use ($db) {
-			$db->create(null);
-		});
-	}
-
-	public function testReadNoConnectionException() {
-		$db = new MongoDb(['host' => '__invalid__', 'autoConnect' => false]);
-		$this->assertException('Could not connect to the database.', function() use ($db) {
-			$db->read(null);
-		});
-	}
-
-	public function testUpdateNoConnectionException() {
-		$db = new MongoDb(['host' => '__invalid__', 'autoConnect' => false]);
-		$this->assertException('Could not connect to the database.', function() use ($db) {
-			$db->update(null);
-		});
-	}
-
-	public function testDeleteNoConnectionException() {
-		$db = new MongoDb(['host' => '__invalid__', 'autoConnect' => false]);
-		$this->assertException('Could not connect to the database.', function() use ($db) {
-			$db->delete(null);
-		});
-	}
-
-	public function testSourcesNoConnectionException() {
-		$db = new MongoDb(['host' => null, 'autoConnect' => false]);
-		$this->assertException('Could not connect to the database.', function() use ($db) {
-			$db->sources(null);
-		});
-	}
-
-	public function testAtomicUpdate() {
```

Comment:
I don't see where else we are testing atomic updates now. Why did this need to go?